examples/step-31/step-31.cc

```diff
@@ -2202,6 +2218,12 @@ int main (int argc, char *argv[])
       Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv,
                                                            numbers::invalid_unsigned_int);
 
+      // This program can only be run in serial. Otherwise, throw an exception.
+      int size;
+      MPI_Comm_size(MPI_COMM_WORLD,&size);
```

Comment:
This should actually be explained somewhere earlier on in the program where we start initializing MPI vectors: that the program can only be run with a single MPI process (and that step-32 is the parallel version). Consequently, all vectors, while of a type that allows parallel decompositions, will always be initialized in a way so that all elements are stored on the current processor, i.e., with an index set that indicates that *every* element is stored locally.